### DISCUSSION TITLE
Custom regex patterns?

### DISCUSSION THREAD

**Discussion by seqizz** [bensadeh/tailspin#102] [Ideas]
Hi :wave: 

First of all, cool project. Tried on few places and liking it.

A small question is: Do you plan to support custom patterns? Something like:
```
[our_special_fqdn]
segment = { fg = "cyan", italic = true }
separator = { fg = "red" }
regexp = ' [a-z0-9-]+.[a-z0-9]+\.business\.local '
```

Here is an example I see on latest release:

![image](https://github.com/bensadeh/tailspin/assets/307899/7e5a1724-5f25-45da-b388-a1ce5247e125)

As far as I understand, this is the "process" but can't catch `udev-worker` due to parentheses.

Considering weird logs we see every day, I thought this might be a nice addition for ones who'd like to lose some sanity via regex and gain amazing customization powers.

**Comment by bensadeh** [comment]
Sorry, I didn't see this until now, but custom regex support has been added to tailspin, hope you find it useful.
